WISE-Community/WISE
# AngularJS/Angular hybrid client-side applications

(This is a branch off of the JSPM-> WebPack PR: #2009 to make the vle/cm/at applications run as hybrid AngularJS/Angular applications. We are no longer using WebPack to build the apps. Instead, we're using Angular-cli.)

How to test:

1. Run npm install
1. Start WISE as before (localhost:8080)
1. Start dev builds for vle/cm/at/site ```npm run build-dev-all```
1. Make sure that all the applications work as before
1. Make changes to applications and test the changes
1. Shutdown dev build process (ctrl-c)
1. Build production applications for vle/cm/at/site ```npm run build-prod-all```
1. Make sure that all the applications work as before

You can also run builds for apps individually ```npm run build-[dev/prod]-[vle/cm/at/site]```. This can save processing and start up times.

Closes #2004
Closes #2022
